packages/react/src/ErrorBoundary.en.mdx

```diff
@@ -48,6 +48,68 @@ const ErrorAfter4s = () => {
 
 ![Example banner](/gif/errorboundary-example.gif)
 
+:::tip
+
+## Define component as `<ErrorBoundary/>`'s fallback
+
+### ErrorBoundaryFallbackProps
+
+If you want to deliver a declared component as `<ErrorBoundary/>`'s fallback, you can use the `ErrorBoundaryFallbackProps` type to declare the component easily.
+
+```tsx
+import type { ErrorBoundaryFallbackProps } from '@suspensive/react'
+
+const ErrorBoundaryFallback = ({ reset, error }: ErrorBoundaryFallbackProps) => (
+  <>
+    <button onClick={reset}>reset</button>
+    {error.message}
+  </>
+)
+
+const Example = () => (
+  <ErrorBoundary fallback={ErrorBoundaryFallback}>
+    <ErrorAfter4s />
+  </ErrorBoundary>
+)
```

Comment:
If it's for the purpose of providing typing

Wouldn't it be better for the developer experience to define not only the type of props but also the return?

```suggestion
import type { ErrorBoundaryFallbackType } from '@suspensive/react'

const ErrorBoundaryFallback: ErrorBoundaryFallbackType = ({ reset, error }) => (
  <>  // The type of return value is also certain
    <button onClick={reset}>reset</button>
    {error.message}
  </>
)

const Example = () => (
  <ErrorBoundary fallback={ErrorBoundaryFallback}> 
    <ErrorAfter4s />
  </ErrorBoundary>
)
```

I inspried by nextjs GetServerSideProps

Reply:
> Wouldn't it be better for the developer experience to define not only the type of props but also the return?

I think that's a good idea too. However, even if we do not provide the return type, TypeScript will create a type error if it does not match the fallback type of ErrorBoundary.

I think we need to consider whether providing a return type is right choice. Because the return type may be different from the actual thing returned ([this video](https://www.youtube.com/watch?v=I6V2FkW1ozQ&ab_channel=Theo-t3%E2%80%A4gg) explain it well). If it is as essential as ErrorBoundaryFallbackProps, it would be a good idea to add it as a minor version feature.

Let's consider it after merging this